test/paper_trail/bang_functions_simple_mode_test.exs

```diff
@@ -978,11 +993,6 @@ defmodule PaperTrailTest.SimpleModeBangFunctions do
     first(Person, :id) |> MultiTenant.add_prefix_to_query() |> repo().one()
   end
 
-  defp serialize(model) do
```

Comment:
This made me like the `PaperTrail.Serializer` module even more, thank you. I'll read up on `Ecto.embedded_dump`